### ISSUE TITLE
Don't know how else to contact you

### ISSUE THREAD

**Issue by inunotaisho** [benawad/slack-clone-client#1]
I noticed that you were using graphql on the frontend and I need a similar setup for an electron app I am working on. Would you be willing to provide me with some knowledge on how to do the same with a NoSQL?

**Comment by benawad** [comment]
graphql on the frontend will be exactly the same no matter what database you use for your graphql api.

**Comment by inunotaisho** [comment]
Has the React-Apollo api changed since your video was created? Not seeing an exported member for `createNetworkInterface` using your original code. Was able to find `ApolloClient` in `apollo-boost`.

**Comment by benawad** [comment]
Yeah here's how you do it now: https://www.apollographql.com/docs/react/get-started/
